packages/namekit-react/src/components/PriceInfo.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { SVGProps } from "react";
+import { Tooltip } from "@namehash/nameguard-react";
+import {
+  getPriceDescription,
+  Registration,
+  DomainName,
+} from "@namehash/ens-utils";
+
+interface PriceInfoProps {
+  parsedName: DomainName;
+  registration: Registration;
+  // Wether to display the icon with a tooltip or the icon + the price info text directly
+  displayIconWithTooltip?: boolean;
+}
+
+export const PriceInfo = ({
```

Comment:
Suggest we make this name more specific. For example maybe `RegistrationPriceInfo`?